core/files/manager.php

```diff
@@ -133,6 +135,36 @@ public function clear_cache() {
 		do_action( 'elementor/core/files/clear_cache' );
 	}
 
+	/**
+	 * Generate all CSS cache or files.
+	 *
+	 * @access public
+	 */
+	public function generate_all_css() {
```

Comment:
Why can't you use `Plugin::$instance->files_manager->clear_cache();`?

Reply:
It doesn't create the new files. the purpose here is for a post-process on the generated files.